### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by agrawalshanu708**
Create Login page
Create Signup page

### CODE REVIEW

File: src/components/Modal/Modal.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.playlistModal{
+    background-color: var(--dark-black);
+    width: 50%;
+    position: absolute;
```
Comment (tanishk15sharma) [inline src/components/Modal/Modal.css:4]: give its parent element `position:relative`

File: src/components/Modal/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import "./Modal.css"
+import {logo} from "../../assets/index"
+import {useState} from "react"
```
Comment (zabihshaik) [inline src/components/Modal/Modal.jsx:4]: Can have this in line no.1 itself

File: src/pages/Auth/signup/signup.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <div className="signup__page">
+    <form action="" className="signup__form">
+        <div className="signup__form__text">Signup</div>
+        <input type="text" placeholder="Name" name="" />
+        <input type="text" placeholder="Email" name="" />
+        <input type="password" placeholder="create password" name="" />
+        <input type="password" placeholder="confirm password" name="" />
+        <input type="submit" value= "Signup" className="button primary_btn"/>
+        <div className="signup__link">
```
Comment (yaswanthmyneni) [inline src/pages/Auth/signup/signup.jsx:14]: changes: give labels to inputs

File: src/pages/Auth/login/login.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+const Login = () => {
+  return (
+    <>
+    <div>
+        <form action="" className="login__form">
+            <div className="login__form__text">Login</div>
+            <input type="text" placeholder="xyz@abc.com" name="" />
+            <input type="password" placeholder="user password" name="" />
+            <input type="submit" value= "Login" className="button primary_btn"/>
+            <div className="signup__link">
```
Comment (yaswanthmyneni) [inline src/pages/Auth/login/login.jsx:13]: same here, add labels to inputs

**Review by zabihshaik — APPROVED** [review]
Everything looks good. Just a comment.

File: src/pages/Auth/login/login.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+            <div className="small_text">Not a member?</div>
+            <Link to = "/signup" ><span className="signup_text">Signup</span></Link> 
+            </div>
+            </form>
+
+
```
Comment (tanishk15sharma) [inline src/pages/Auth/login/login.jsx:19]: use `prettier`

File: src/pages/Auth/login/login.jsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import "./login.css"
+import {Link} from "react-router-dom"
+const Login = () => {
+  return (
+    <>
+    <div>
+        <form action="" className="login__form">
+            <div className="login__form__text">Login</div>
+            <input type="text" placeholder="xyz@abc.com" name="" />
+            <input type="password" placeholder="user password" name="" />
```
Comment (tanishk15sharma) [inline src/pages/Auth/login/login.jsx:11]: give `name` attribute to both the `input`

Comment (yaswanthmyneni) [inline src/pages/Auth/login/login.jsx:11]: changes: remove name attribute if it an empty string

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after changes.

File: src/pages/Auth/signup/signup.jsx [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+import "./signup.css"
+const Signup = () => {
+  return (
+    <div className="signup__page">
+    <form action="" className="signup__form">
+        <div className="signup__form__text">Signup</div>
+        <input type="text" placeholder="Name" name="" />
+        <input type="text" placeholder="Email" name="" />
+        <input type="password" placeholder="create password" name="" />
+        <input type="password" placeholder="confirm password" name="" />
```
Comment (tanishk15sharma) [inline src/pages/Auth/signup/signup.jsx:12]: Don't leave `name`  attribute empty .

File: src/App.js [hunk 8]
```diff
@@ -2,7 +2,7 @@ import "./App.css";
 import logo from "./logo.png";
 import { Routes, Route, Link } from "react-router-dom";
 import {Navbar,Searchbar} from "./components/index"
-import {Home,LikePage,WatchLaterPage,HistoryPage} from "./pages/index"
+import {Home,LikePage,WatchLaterPage,HistoryPage,Playlist,Login,Signup} from "./pages/index"
```
Comment (ravikumar0403) [inline src/App.js:5]: you can omit `/index` here

**Review by yashpatel1797 — APPROVED** [review]
code looks clean
file structure is good
design looks consistent

File: src/pages/playlistpage/playlist.jsx [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import "./playlist.css"
+const Playlist = () => {
+  return (
+     <div className="playlist-page">
+     <button className="btn btn-primary"
+     onClick = {() => {}}
```
Comment (tanishk15sharma) [inline src/pages/playlistpage/playlist.jsx:7]: you are having nothing in `onClick`

File: src/backend/db/videos.js [hunk 10]
```diff
@@ -9,35 +9,40 @@ export const videos = [
   title: "talks",
   charactor: "sunadar pichai",
   description: "",
+  modal: false,
   link: "https://www.youtube.com/watch?v=-yLlMk41sro"
 },{
   _id: "Lb4IcGF5iTQ",
   title: "talks",
   charactor: "Mark Zuckerberg",
   description: "",
```
Comment (ckahub) [inline src/backend/db/videos.js:18]: u have mention description at some places and missed at some

File: src/components/Modal/Modal.css [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+    height: 45%;
+    z-index: 99;
+    border-radius: 1rem;
+    padding: 2rem 10rem;
+}
+.hr{
+    border-top: 1px solid var(--primary-color);
+   margin: 1rem 0
+}
+.playlistModal input{
```
Comment (ravikumar0403) [inline src/components/Modal/Modal.css:17]: try not to use html tags directly for styling

File: src/backend/db/videos.js [hunk 12]
```diff
@@ -9,35 +9,40 @@ export const videos = [
   title: "talks",
   charactor: "sunadar pichai",
   description: "",
```
Comment (ckahub) [inline src/backend/db/videos.js:11]: same as below

File: src/components/Modal/Modal.jsx [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+    </div> 
+    <div className="align__vertical">
+    <label htmlFor="playlist__desc" className="playlist__form__label">Description</label>
+    <textArea id="playlist__desc" className="playlist__form__textarea"></textArea>
+    </div>
+    <div className="align__horizontal">
+      <button className="button primary_btn" type="submit">Create Collection</button>
+      <button className="button secondary_btn">Cancel</button>
+      </div>
+    </form>
```
Comment (ravikumar0403) [inline src/components/Modal/Modal.jsx:26]: use prettier to format your code

File: src/components/videostore/videocard.jsx [hunk 14]
```diff
@@ -1,73 +1,112 @@
...
+
+  return (
+    <div class="border-skin text-overlay-card-dimension card-relative video-card" key = {_id}>
+      <div
+        class="text-overlay-card-img-box"
+        onClick={() => historyHandler(_id, product)}
+      >
+        <img src={`https://i.ytimg.com/vi/${_id}/hqdefault.jpg`} />
       </div>
       {/* <div class="card-head head-overflow">{title}</div> */}
```
Comment (ckahub) [inline src/components/videostore/videocard.jsx:72]: remove comment code

**Review by tanishk15sharma — APPROVED** [review]
I have suggested some changes above , use any code editor for your code it will be good to review .

File: src/components/navbar/navbar.jsx [hunk 15]
```diff
@@ -38,7 +38,7 @@ const Navbar = () => {
 
 </div>
 <div className="navbar__profile">
-    <span className="navbar__text">Login</span>
+  <Link to = "./login"> <span className="navbar__text">Login</span></Link>
```
Comment (ravikumar0403) [inline src/components/navbar/navbar.jsx:41]: it should be `/login` instead of `./login`

File: src/components/videostore/videocard.jsx (hunk 14 above)
Comment (ravikumar0403) [inline src/components/videostore/videocard.jsx:72]: remove commented code if not required

**Review by ravikumar0403 — APPROVED** [review]
Try to use some code formatting extension to format you code

**Review by harshitpaliwal95 — COMMENTED** [review]
everything looks do suggested change and good to go

**Pull request merged**